packages/fkweb/fk/migrations/0002_rename_all_custom_tables_to_default.py

```diff
@@ -24,20 +30,22 @@ class Migration(migrations.Migration):
             name='fileformat',
             table=None,
         ),
+        # By calling RenameModel, for some reason Django no longer
+        # fails to update the ForeignKey references in the schema.
+        migrations.RenameModel('Organization', 'OrganizationNew'),
         migrations.AlterModelTable(
-            name='organization',
```

Comment:
This migration has already been run for the servers. You should make a new migration with all this in it, a 0006 if you will.

That one will also be run on the production servers then.

You can test by pushing to dev. -- Though if it fails, you have to clean the database up manual...yyy. Maybe not actually. Postgresql has transactions on migrations too. So should be okay. Anyway, easy to test on dev, and put in the new database if it went bad.